### PR TITLE
fix/remove-error-polyfill unused dependecy that breaks modern build tools

### LINKED ISSUE #688: URGENT fix broken dependency please
**Describe the bug**

NAJ relies on:
https://www.npmjs.com/package/error-polyfill

which relies on unmaintained:
https://www.npmjs.com/package/u3

**which has the bug:**
https://github.com/inf3rno/u3/issues/3

when bundling with modern bundlers e.g. parcel 2:
https://github.com/parcel-bundler/parcel/releases/tag/v2.0.0-rc.0

**To Reproduce**
Try to bundle NAJ with 
https://github.com/parcel-bundler/parcel/releases/tag/v2.0.0-rc.0

**Expected behavior**
It works.

**More info**
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Delete_in_strict_mode

**Desktop (please complete the following information):**
 - Ubuntu 20.04

**Additional context**
URGENT please, breaks all my builds and I have to patch this deep dep across every repo

### PR DESCRIPTION
**Pull request by mattlockyer**
Fixes #688

### CODE REVIEW

**Comment by mattlockyer** [comment]
@chadoh tests pass :+1:

**Comment by mikedotexe** [comment]
Pinging Vlad on this now to see if he recalls what happened in this line 9 months ago:
https://github.com/near/near-api-js/blame/e29a41812ac79579cc12b051f8ef04d2f3606a75/src/browser-index.ts#L6

**Comment by mattlockyer** [comment]
Resolved. Can confirm Parcel 2 builds working now!

Bump error-polyfill version which bumps u3 version.

inf3rno/error-polyfill@d6286da

and

inf3rno/u3@61af796

Thanks to https://github.com/inf3rno

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
         "borsh": "^0.6.0",
         "bs58": "^4.0.0",
         "depd": "^2.0.0",
-        "error-polyfill": "^0.1.2",
+		"error-polyfill": "^0.1.3",
```
Comment (volovyk-s) [inline package.json:18]: Looks like the formatting is broken.

**Review by chadoh — APPROVED** [review]
Thanks!

**Comment by volovyk-s** [comment]
@mattlockyer thank you for this fix!

**Pull request merged**